tests/test_ensemble_grant_tagger.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import tempfile
+import os
+
+import pandas as pd
+
+from nutrition_labels.ensemble_grant_tagger import EnsembleGrantTagger
+from nutrition_labels.grant_tagger import GrantTagger
+
+# Designed to give varying results for count and tfidf
+text_data = pd.DataFrame(
+            [
+                {'id':0, 'text_1': 'dog dog', 'text_2': '', 'label': 1},
+                {'id':1, 'text_1': 'dog dog', 'text_2': '', 'label': 1},
+                {'id':2, 'text_1': 'dog skate', 'text_2': '', 'label': 0},
+                {'id':3, 'text_1': 'cat shoe', 'text_2': '', 'label': 0},
+                {'id':4, 'text_1': 'dog dog skate cat shoe sandal', 'text_2': '', 'label': 0},
+                {'id':5, 'text_1': 'sandal', 'text_2': '', 'label': 0},
+                {'id':6, 'text_1': None, 'text_2': None, 'label': 0}
+            ]
+        )
+
+def test_load_grants_text():
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
I think pytest has a tmp_path fixture that is provided automatically so if you do `test_load_grants_text(tmp_path)` you can remove this line

see https://docs.pytest.org/en/stable/tmpdir.html

Reply:
whoa 🤯